gui/cache.go

```diff
@@ -94,6 +95,13 @@ func (c *Cache) getMinedWork() []minedWork {
 
 // updateQuotas refreshes the cached list of pending dividend payments.
 func (c *Cache) updateQuotas(quotas []*pool.Quota) {
+
+	// Sort list so the largest percentages will be shown first.
+	sort.Slice(quotas, func(i, j int) bool {
+		larger := quotas[i].Percentage.Cmp(quotas[j].Percentage) > 0
```

Comment:
simpler to do `return quotas[i].Percentage.Cmp(quotas[j].Percentage) > 0`

Reply:
agreed